src/client/camera.h

```diff
@@ -48,6 +48,10 @@ struct Nametag {
 	v3f nametag_pos;
 };
 
+struct CameraModeRestrictions {
+	bool first, third, third_front;
```

Comment:
initialize values in the definition please

Reply:
Done; also added a ctor.